lib/src/js/value/color.dart

```diff
@@ -86,4 +447,30 @@ class _Channels {
   external double? get whiteness;
   external double? get blackness;
   external double? get alpha;
+  external double? get a;
+  external double? get b;
+  external double? get x;
+  external double? get y;
+  external double? get z;
+  external double? get chroma;
+}
+
+@JS()
+@anonymous
+class _ConstructionOptions extends _Channels {
+  external String? get space;
+}
+
+@JS()
+@anonymous
+class ChannelOptions {
+  String? space;
+}
+
+@JS()
+@anonymous
+class InterpolationOptions {
+  external SassColor color2;
```

Comment:
It seems like `color2` should probably be a positional parameter, not an option. Sorry, I should have caught that during API review.

Reply:
Adjusted in https://github.com/oddbird/dart-sass/commit/cfeb456e1bb29ca6400c2972aa8085b75673cc54 and https://github.com/oddbird/sass/commit/424b0d295623c77b5e04c82d8f2e4a59f6a6f492.